src/content/pages/pyladies-events.mdx

```diff
@@ -6,17 +6,50 @@ subtitle: Lunch, Workshops & Networking
 # PyLadies Events
 We’re excited to announce a range of events for underrepresented groups in computing this year! 🎉 Whether you're new to PyLadies or a long-time supporter, we warmly welcome you to join us and be part of our supportive community.
 
-## #IAmRemarkable
-Empower yourself in this two-hour workshop designed to help you celebrate your achievements and improve your self-promotion skills.
+![](./images/pyladies-events.jpg)
+
+## Self-defence Workshop
+A two-hours workshop where you can share your challenging experiences and learn how to defend yourself against inappropriate behaviour. This is a space to foster mutual support, understanding and effective strategies for personal safety. This session will be facilitated by a therapist.
+
+WHEN?
+Thursday, 11th July 2024, 10:30 to 12:30.
+
+WHERE?
+Prague Congress Centre (PCC), Open Space 2 (Rooms 223 + 224)
 
 ## PyLadies Lunch
 Join us for a special lunch event aimed at fostering community and empowerment in tech. Enjoy meaningful conversations and networking opportunities.
```

Comment:
We should mention the Sponsor for the lunch here. Something like:

_"Thank you to our sponsor 🐙 Kraken Tech 🐙 for supporting the lunch event."_

Feel free to reword it, though.

Reply:
you're right, I forgot about the sponsors. text added ✅